node/http.go

```diff
@@ -74,7 +84,7 @@ func makeRPCOverHTTP(host string, port uint16, servers map[string]*jsonrpc.Serve
 		if listener != nil {
 			httpHandler = httpHandler.WithListener(listener)
 		}
-		mux.Handle(path, httpHandler)
+		mux.Handle(path, exactPathServer(path, httpHandler))
```

Comment:
hm.... I always thought that `Handle(...)` use strict matching at least for `/path?query=` or you added it so it won't be possible to pass get parameters?

Reply:
> I always thought that Handle(...) use strict matching

sadly it doesnt, "/" matches every possible path.

Reply:
https://pkg.go.dev/net/http#ServeMux

> Note that since a pattern ending in a slash names a rooted subtree, the pattern "/" matches all paths not matched by other registered patterns, not just the URL with Path == "/".